spec/model/answer/money_spec.rb

```diff
@@ -49,30 +49,63 @@
         expect(instance.humanize).to eql("£1,523.42")
       end
     end
+
     context "no pence" do
       let(:money_input) { '1523.00' }
       it "rounds down amounts of money correctly" do
         expect(instance.humanize).to eql("£1,523")
       end
     end
+
+    context "for value with a trailing dot" do
+      let(:money_input) { '1523.' }
+      it "ignores the trailing dot" do
+        expect(instance.humanize).to eql("£1,523")
+      end
+    end
   end
 
-  describe "errors" do
-    context "invalid formatting" do
+  describe "parsing" do
+    context "no digits" do
       let(:money_input) {"Loads'a'money"}
 
-      it "Has errors" do
+      it "raises an error" do
         expect(instance.error).to eql("Invalid format")
       end
     end
 
-    context "no input" do
+    context "empty input" do
       let(:money_input) { nil }
 
-      it "Has errors" do
+      it "raises an error" do
         expect(instance.error).to eql("Please answer this question")
       end
     end
+
+    context "float with a trailing dot" do
+      let(:money_input) { "200." }
+
+      it "raises no errors" do
+        expect(instance.error).to eql(nil)
```

Comment:
The only two public methods I see are #humanise and #to_s, I've added a test for the changed behaviour on #humanise (see line 59 above). However the parsing logic is invoked from the [initializer](https://github.com/alphagov/smartdown/blob/tt_forgiving_floats/lib/smartdown/model/answer/base.rb#L41) (I'm generally not a fan of any logic being invoked in the initializer) so I wanted to make sure it's tested here too i.e. what if humanize method is deleted together with the test in question?